News.md

```diff
@@ -0,0 +1,5 @@
+# DataFramesMeta v0.6 Release Notes
```

Comment:
Isn't it more common to call this file NEWS.md in uppercase?

Reply:
added.